envi/archs/amd64/disasm.py

```diff
@@ -238,25 +281,21 @@ def disasm(self, bytez, offset, va):
             if p is None:
                 break
 
-            # print("OBYTE",hex(obyte))
-            if obyte in mandatory_prefixes:
+            if MANDATORY_PREFIXES[obyte]:
```

Comment:
nice!  a pre-genned lookup table which avoids an actual loop the size of MANDATORY_PREFIXES.  costs little memory, and saves on cycles.

Reply:
Aye. Shamelessly ripped from some of the other pregenned tables we've got lying around :) . Helped a ton with performance, and since I replicated this over into i386, we should see a bit of a bump there in terms of performance.